packages/lib/src/vis/shared/AxisSystem.module.css

```diff
@@ -1,4 +1,6 @@
 .axisSystem {
+  position: absolute;
+  pointer-events: none;
```

Comment:
`AxisSystem` no longer uses `Overlay`. I could have forwarded the `container` prop but this seemed overkill given that `AxisSystem` already declares most of `Overlay`'s CSS properties (`top`, `width`, etc.)